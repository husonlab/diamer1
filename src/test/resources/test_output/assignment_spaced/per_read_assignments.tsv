6
ReadID	OVO (0.20) read count	OVO (0.20) read count (norm. kmers)	OVO (0.50) read count	OVO (0.50) read count (norm. kmers)	OVO (0.60) read count	OVO (0.60) read count (norm. kmers)	OVO (0.80) read count	OVO (0.80) read count (norm. kmers)	OVO (0.90) read count	OVO (0.90) read count (norm. kmers)	OVO (1.00) read count	OVO (1.00) read count (norm. kmers)	OVA (0.70) read count	OVA (0.70) read count (norm. kmers)
0	(1 from root) rank2 (2)	(1 from root) rank2 (2)	(1 from root) rank2 (2)	(1 from root) rank2 (2)	(1 from root) rank2 (2)	(1 from root) rank2 (2)	(1 from root) rank2 (2)	(1 from root) rank2 (2)	(1 from root) rank2 (2)	(1 from root) rank2 (2)	(1 from root) rank2 (2)	(1 from root) rank2 (2)	(1 from root) rank2 (2)	(1 from root) rank2 (2)
1														
2														
3														
4	(1 from root) rank1 (1)	(1 from root) rank1 (1)	(1 from root) rank1 (1)	(1 from root) rank1 (1)	(1 from root) rank1 (1)	(1 from root) rank1 (1)	(1 from root) rank1 (1)	(1 from root) rank1 (1)	(1 from root) rank1 (1)	(1 from root) rank1 (1)	(1 from root) rank1 (1)	(1 from root) rank1 (1)	(1 from root) rank1 (1)	(1 from root) rank1 (1)
5														
