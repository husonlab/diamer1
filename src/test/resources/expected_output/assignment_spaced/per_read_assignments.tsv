6
ReadID	OVO (ratio: 0.200000)	OVO (ratio: 0.200000)	OVO (ratio: 0.800000)	OVO (ratio: 0.800000)
0	(1 from root) rank2 (2)	(1 from root) rank2 (2)	(1 from root) rank2 (2)	(1 from root) rank2 (2)
1				
2				
3				
4	(1 from root) rank1 (1)	(1 from root) rank1 (1)	(1 from root) rank1 (1)	(1 from root) rank1 (1)
5				
